### PR TITLE
feat: Create media page

### PR DESCRIPTION
**Pull request by Sanskarpanchal27**

### CODE REVIEW

File: src/Pages/Media/Carousel.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <button
+          className="carousel-control-next"
+          type="button"
+          data-bs-target={`#${id}`}
+          data-bs-slide="next"
+        >
+          <span
+            className="carousel-control-next-icon"
+            aria-hidden="true"
+          ></span>
```
Comment (vaidehi27) [inline src/Pages/Media/Carousel.tsx]:
```suggestion
          <span
            className="carousel-control-next-icon"
            aria-hidden="true"
          />
```

File: src/Pages/Media/Carousel.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <button
+          className="carousel-control-prev"
+          type="button"
+          data-bs-target={`#${id}`}
+          data-bs-slide="prev"
+        >
+          <span
+            className="carousel-control-prev-icon"
+            aria-hidden="true"
+          ></span>
```
Comment (vaidehi27) [inline src/Pages/Media/Carousel.tsx]:
```suggestion
          <span
            className="carousel-control-prev-icon"
            aria-hidden="true"
          />
```

File: src/Pages/Media/constant.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import sringar1 from "../../assets/images/sringar1.jpg";
+import sringar2 from "../../assets/images/sringar2.jpg";
+import sringar3 from "../../assets/images/sringar3.jpg";
+import darshan1 from "../../assets/images/darshan1.jpg";
+import darshan2 from "../../assets/images/darshan2.jpg";
+import darshan3 from "../../assets/images/darshan3.jpg";
+
+export const sringar = [sringar1, sringar2, sringar3];
```
Comment (vaidehi27) [inline src/Pages/Media/constant.ts]:
```suggestion
export const sringarImages = [sringar1, sringar2, sringar3];
```

File: src/Pages/Media/constant.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+export const darshan = [darshan1, darshan2, darshan3];
```
Comment (vaidehi27) [inline src/Pages/Media/constant.ts]:
```suggestion
export const darshanImages = [darshan1, darshan2, darshan3];
```

File: src/Pages/Media/style.ts [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+`;
+
+export const LinkName = styled.p`
+  font-size: 2.5rem;
+  color: red;
+  text-align: center;
+  font-family: Times New Roman;
+`;
+
+export const ImageDiv = styled.div`
```
Comment (vaidehi27) [inline src/Pages/Media/style.ts]:
```suggestion
export const ImageWrapper = styled.div`
```

File: src/Pages/Media/Carousel.tsx (hunk 2 above)
Comment (Sanskarpanchal27) [reply src/Pages/Media/Carousel.tsx]: changed done

File: src/Pages/Media/constant.ts (hunk 3 above)
Comment (Sanskarpanchal27) [reply src/Pages/Media/constant.ts]: changed done

File: src/Pages/Media/style.ts (hunk 5 above)
Comment (Sanskarpanchal27) [reply src/Pages/Media/style.ts]: changed done

File: src/Pages/Media/constant.ts (hunk 4 above)
Comment (Sanskarpanchal27) [reply src/Pages/Media/constant.ts]: changed done

File: src/Pages/Media/Audio.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    <div>
+      <a href="https://drive.google.com/" title="Google Drive Link">
+        <LinkName>Google Drive Link</LinkName>
+      </a>
+      <Heading>Kirtan</Heading>
+      <ImageWrapper>
+        <TitleImage src={titleBaseImg} alt="TitleBase" />
+      </ImageWrapper>
+      {/* <br />
+      <br /> */}
```
Comment (vaidehi27) [inline src/Pages/Media/Audio.tsx]:
```suggestion
```

File: src/Pages/Media/Audio.tsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+      <IframeWrapper>
+        <iframe
+          title="Spotify Playlist"
+          src="https://open.spotify.com/embed/playlist/3ZPZj8PSYOIyJmHpEI4YLK?utm_source=generator"
+          width="352"
+          height="400"
+          allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"
+          loading="lazy"
+        ></iframe>
```
Comment (vaidehi27) [inline src/Pages/Media/Audio.tsx]:
```suggestion
        <iframe
          title="Spotify Playlist"
          src="https://open.spotify.com/embed/playlist/3ZPZj8PSYOIyJmHpEI4YLK?utm_source=generator"
          width="352"
          height="400"
          allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"
          loading="lazy"
        />
```

File: src/Pages/Media/Carousel.tsx [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+      id={id}
+      className="carousel slide"
+      data-bs-ride="carousel"
+      data-bs-interval="2000"
+    >
+      <div className="carousel-inner">
+        {images.map((image) => (
+          <div
+            key={Math.random()}
+            className={`carousel-item ${image === images[0] ? "active" : ""}`}
```
Comment (vaidehi27) [inline src/Pages/Media/Carousel.tsx]:
```suggestion
        {images.map((image, index) => (
          <div
            key={Math.random()}
            className={`carousel-item ${index === 0 ? "active" : ""}`}
```

File: src/Pages/Media/index.tsx [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+        <CarouselWrapper>
+          <Carousel images={sringarImages} />
+          <TitleName>Yesterday's Darshan </TitleName>
+        </CarouselWrapper>
+        <CarouselWrapper>
+          <Carousel images={darshanImages} />
+          <TitleName>Today's Darshan </TitleName>
+        </CarouselWrapper>
+      </PageWrapper>
+      <>
```
Comment (vaidehi27) [inline src/Pages/Media/index.tsx:34]: Is this empty tag needed?

File: src/Pages/Media/youtube.tsx [hunk 10]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import YouTube from "react-youtube";
+
+interface Props {
+  videoId: string;
+}
+
+export const YouTubeVideo: React.FC<Props> = ({ videoId }) => {
```
Comment (vaidehi27) [inline src/Pages/Media/youtube.tsx]: is this component really needed?

Comment (vaidehi27) [reply src/Pages/Media/youtube.tsx]: It does nothing. Can we directly use `YouTube` component?

**Review by vaidehi27 — APPROVED** [review]

**Pull request merged**